tests/test_visitors/test_tokenize/test_statements/test_inconsistent_comprehension.py

```diff
@@ -11,26 +11,61 @@
 
 # List comprehension tests
 
-correct_list_empty = """
-a = []
+correct_list_empty = '[]'
+
+correct_list_one_line_comprehension1 = """
+def wrapper():
```

Comment:
why it's necessary to add `def wrapper():`?
Maybe it can be added in the test to make test cases declaration less verbose?

Reply:
See `mode` fixture: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/plugins/async_sync.py